checkstyle-tester/launch.groovy

```diff
@@ -313,6 +313,7 @@ def removeEmptyDirectories(file) {
 }
 
 def executeCmd(cmd, dir =  new File("").getAbsoluteFile()) {
+    println cmd
```

Comment:
Should we add some extra text so it is clear this is the command being executed?
Ex: "Running Command: ...."